File: run-with-me/drools-spring-app/src/main/java/com/app/rwm/model/Injury.java
package com.app.rwm.model;

import java.util.Date;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

import com.app.rwm.enums.INJURY_TYPE;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Entity
public class Injury {
	
	 @Id
	 @GeneratedValue(strategy = GenerationType.IDENTITY)
	 private Long id;
	 
	 private Date dateOfInjury;
	 private INJURY_TYPE injuryType;
	 
}

File: run/rwm-front/src/app/core/model/injury.ts
export class Injury {
    username: String;
    dateOfInjury : any;
    injuryType: string;

    constructor(username : String, dateOfInjury : any, injuryType : string){
        this.username = username;
        this.dateOfInjury = dateOfInjury;
        this.injuryType = injuryType;
    }

}
